### PR TITLE
2 step transfer ownership - IO1-434

### PR DESCRIPTION
**Pull request by mtabasco**
Use OZ's `Ownable2StepUpgradeable` to support 2-step transfer ownership functionality

### CODE REVIEW

File: test/dao/operational.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+let ssvNetworkContract: any;
+
+// Declare globals
+describe('DAO operational Tests', () => {
+    beforeEach(async () => {
+        // Initialize contract
+        ssvNetworkContract = (await helpers.initializeContract()).contract;
+    });
+
+    it('starting the transfer process does not change owner', async () => {
```
Comment (mtabasco) [inline test/dao/operational.ts:14]: uppercase and format.
add one test about change onlyOwner variable and try to change with the old owner

File: scripts/upgrade-ssv-network-views.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-const { ethers, upgrades } = require("hardhat");
+import { ethers, upgrades } from 'hardhat';
 
 async function upgradeSSVNetworkViews() {
-  const proxyAddress = process.env.SSVNETWORKVIEWS_PROXY_ADDRESS;
+  const proxyAddress: any = process.env.SSVNETWORKVIEWS_PROXY_ADDRESS;
```
Comment (andrew-blox) [inline scripts/upgrade-ssv-network-views.ts:4]: Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)

File: scripts/upgrade-ssv-network.ts [hunk 3]
```diff
@@ -1,5 +1,7 @@
+import { ethers, upgrades } from 'hardhat';
+
 async function upgradeSSVNetwork() {
-  const proxyAddress = process.env.SSVNETWORK_PROXY_ADDRESS;
+  const proxyAddress: any = process.env.SSVNETWORK_PROXY_ADDRESS;
```
Comment (andrew-blox) [inline scripts/upgrade-ssv-network.ts:4]: Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)

File: scripts/validate-upgrade-ssv-network-views.ts [hunk 4]
```diff
@@ -1,5 +1,7 @@
+import { ethers, upgrades } from 'hardhat';
+
 async function validateUpgradeSSVNetworkViews() {
-  const proxyAddress = process.env.SSVNETWORKVIEWS_PROXY_ADDRESS;
+  const proxyAddress: any = process.env.SSVNETWORKVIEWS_PROXY_ADDRESS;
```
Comment (andrew-blox) [inline scripts/validate-upgrade-ssv-network-views.ts:4]: Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)

File: scripts/validate-upgrade-ssv-network.ts [hunk 5]
```diff
@@ -1,6 +1,8 @@
+import { ethers, upgrades } from 'hardhat';
+
 async function validateUpgradeSSVNetwork() {
-  const proxyAddress = process.env.SSVNETWORK_PROXY_ADDRESS;
-  const SSVNetwork = await ethers.getContractFactory("SSVNetwork_V2");
+  const proxyAddress: any = process.env.SSVNETWORK_PROXY_ADDRESS;
```
Comment (andrew-blox) [inline scripts/validate-upgrade-ssv-network.ts:4]: Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)

File: test/dao/operational.ts [hunk 6]
```diff
@@ -0,0 +1,27 @@
+// Declare imports
+import * as helpers from '../helpers/contract-helpers';
+import { expect } from 'chai';
+
+let ssvNetworkContract: any;
```
Comment (andrew-blox) [inline test/dao/operational.ts:5]: Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)

File: README.md [hunk 7]
```diff
@@ -199,6 +192,7 @@ To validate the upgrade before running it:
 ```sh
 npx hardhat run --network <your-network> scripts/validate-upgrade-ssv-network.ts
 ```
+Like on the deployment process, you will be guided to the steps for creating or reusing a git tag for the new upgrade.
```
Comment (andrew-blox) [inline README.md:195]: remove "on"

"Like the deployment process"

File: README.md [hunk 8]
```diff
@@ -228,6 +222,7 @@ To validate the upgrade before running it:
 ```sh
 npx hardhat run --network <your-network> scripts/validate-upgrade-ssv-network-views.ts
 ```
+Like on the deployment process, you will be guided to the steps for creating or reusing a git tag for the new upgrade.
```
Comment (andrew-blox) [inline README.md:225]: remove "on"

"Like the deployment process"

**Review by andrew-blox — CHANGES_REQUESTED** [review]

**Review by lior-blox — APPROVED** [review]

**Review by andrew-blox — APPROVED** [review]

**Pull request merged**